Arduino/MiP_ProMini_Pack_Library/src/mip.cpp

```diff
@@ -80,6 +80,8 @@
 #define MIP_CMD_FLASH_CHEST_LED         0x89
 #define MIP_CMD_SET_HEAD_LEDS           0x8A
 #define MIP_CMD_GET_HEAD_LEDS           0x8B
+#define MIP_CMD_DISCONNECT_APP			0xFE
```

Comment:
Can you setup your text editor to use spaces instead of tabs so that your indentation is explicit and you don't see these weird alignments in the diff?

Reply:
Done.  Using Notepad++.  What are you using?